include/libtorrent/kademlia/routing_table.hpp

```diff
@@ -73,6 +73,29 @@ struct routing_table_node
 	bucket_t live_nodes;
 };
 
+struct ip_set
```

Comment:
it would be nice to have a unit test for this. you could make the class TORRENT_EXTRA_EXPORT.